src/ol/source/stamen.js

```diff
@@ -35,7 +35,8 @@ ol.source.Stamen = function(options) {
     opaque: layerConfig.opaque,
     reprojectionErrorThreshold: options.reprojectionErrorThreshold,
     tileLoadFunction: options.tileLoadFunction,
-    url: url
+    url: url,
+    wrapX: options.wrapX
```

Comment:
See https://github.com/openlayers/openlayers/blob/master/src/ol/source/xyz.js#L57 (parent class)

If the option is omitted (`undefined`) the value will be `true`